### ISSUE TITLE
sed generates a false positive

### ISSUE THREAD

**Issue by junghans** [koalaman/shellcheck#64]
``` bash
sed '$p' file
```

will generate a false positive.
